__react/utils/bindings.js

```diff
@@ -0,0 +1,79 @@
+export function includeUnsubAfterForSubscription(message) {
+  if (hasNatsBindings(message) && message.bindings().nats().unsubAfter()) {
+    return `subscribeOptions.max = '${message.bindings().nats().unsubAfter()}';`;
+  }
+  return '';
+}
+export function includeQueueForSubscription(message) {
+  if (hasNatsBindings(message) && message.bindings().nats().queue()) {
+    return `subscribeOptions.queue = '${message.bindings().nats().queue()}';`;
+  }
+  return '';
+}
+
+/**
+ * Does an object have bindings
+ */
+export function hasNatsBindings(obj) {
+  return obj.bindings && obj.bindings.nats;
+}
+  
+/**
+   * is the channel a publish and subscribe type if nothing is specified default to being pubsub type 
+   */
```

Comment:
Two things:

1. Indentation.
2. I don't understand this explanation 😄 I have the feeling there's some punctuation mark missing.

Reply:
I have added a new definition :)